<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>740</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../icons.qrc">
    <normaloff>:/icons/resources/icone2.ico</normaloff>:/icons/resources/icone2.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>1</number>
    </property>
    <property name="leftMargin">
     <number>1</number>
    </property>
    <property name="topMargin">
     <number>1</number>
    </property>
    <property name="rightMargin">
     <number>1</number>
    </property>
    <property name="bottomMargin">
     <number>1</number>
    </property>
    <item>
     <widget class="QLineEdit" name="addressLine"/>
    </item>
    <item>
     <widget class="QWebView" name="webView" native="true">
      <property name="url" stdset="0">
       <url>
        <string>about:blank</string>
       </url>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::TopToolBarArea</set>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionExit"/>
   <addaction name="actionSave"/>
   <addaction name="actionPrint"/>
   <addaction name="actionBackward"/>
   <addaction name="actionForward"/>
   <addaction name="actionGo_Home"/>
   <addaction name="actionStop"/>
   <addaction name="actionReload"/>
   <addaction name="actionGo"/>
  </widget>
  <action name="actionBackward">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/resources/bacward.png</normaloff>:/icons/resources/bacward.png</iconset>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
   <property name="toolTip">
    <string>Back</string>
   </property>
   <property name="shortcut">
    <string>Backspace</string>
   </property>
  </action>
  <action name="actionForward">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/resources/forward.png</normaloff>:/icons/resources/forward.png</iconset>
   </property>
   <property name="text">
    <string>Forward</string>
   </property>
   <property name="toolTip">
    <string>Forward</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Right</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/resources/stop.png</normaloff>:/icons/resources/stop.png</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionReload">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/resources/reload.png</normaloff>:/icons/resources/reload.png</iconset>
   </property>
   <property name="text">
    <string>Reload</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionGo_Home">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/resources/home.png</normaloff>:/icons/resources/home.png</iconset>
   </property>
   <property name="text">
    <string>Go Home</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/resources/exit.png</normaloff>:/icons/resources/exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/resources/save.png</normaloff>:/icons/resources/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/resources/print.png</normaloff>:/icons/resources/print.png</iconset>
   </property>
   <property name="text">
    <string>Print</string>
   </property>
  </action>
  <action name="actionGo">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/resources/go.png</normaloff>:/icons/resources/go.png</iconset>
   </property>
   <property name="text">
    <string>Go</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKitWidgets/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>webView</sender>
   <signal>loadStarted()</signal>
   <receiver>MainWindow</receiver>
   <slot>loadStarted()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>318</x>
     <y>221</y>
    </hint>
    <hint type="destinationlabel">
     <x>318</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>webView</sender>
   <signal>loadFinished(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>loadFinished(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>318</x>
     <y>221</y>
    </hint>
    <hint type="destinationlabel">
     <x>318</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addressLine</sender>
   <signal>returnPressed()</signal>
   <receiver>actionGo</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>318</x>
     <y>44</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionGo</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>openPage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>318</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionReload</sender>
   <signal>triggered()</signal>
   <receiver>webView</receiver>
   <slot>reload()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>318</x>
     <y>227</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionForward</sender>
   <signal>triggered()</signal>
   <receiver>webView</receiver>
   <slot>forward()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>318</x>
     <y>227</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionBackward</sender>
   <signal>triggered()</signal>
   <receiver>webView</receiver>
   <slot>back()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>318</x>
     <y>227</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSave</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>savePage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>318</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionPrint</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>printPage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>318</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>318</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionGo_Home</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>openHomePage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>318</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>webView</sender>
   <signal>titleChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>titleChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>499</x>
     <y>397</y>
    </hint>
    <hint type="destinationlabel">
     <x>499</x>
     <y>369</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>loadFinished(bool)</slot>
  <slot>loadStarted()</slot>
  <slot>openPage()</slot>
  <slot>goBacward()</slot>
  <slot>goForward()</slot>
  <slot>openHomePage()</slot>
  <slot>savePage()</slot>
  <slot>printPage()</slot>
 </slots>
</ui>
